common/defaults.go

```diff
@@ -231,6 +231,12 @@ const (
 	// ArgoCDDefaultResourceInclusions is the default resource inclusions.
 	ArgoCDDefaultResourceInclusions = ""
 
+	// ArgoCDRolloutExtensionImage is the default image for ArgoCD Rollout Extension Installer
```

Comment:
The name of the controller is `Argo Rollouts`. So, we should change this to `Argo Rollouts`.